ngraph/core/include/ngraph/op/round.hpp

```diff
@@ -90,6 +90,9 @@ namespace ngraph
                 virtual std::shared_ptr<Node>
                     clone_with_new_inputs(const OutputVector& new_args) const override;
 
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Please measure the impact on binary size.

Reply:
7 748 608 bytes -> 7 752 704 bytes

Reply:
this operation is required to have `evaluate` because it appears on a shape calculation paths.